Exiled.API/Features/Player.cs

```diff
@@ -1536,8 +1536,8 @@ public ReadOnlyCollection<PlayerEffect> GetAllEffects()
             ReadOnlyCollection<PlayerEffect> ActiveEffects = new ReadOnlyCollection<PlayerEffect>();
             foreach (var effect in ReferenceHub.playerEffectsController.AllEffects)
             {
-              if (GetEffect((EffectType)effect.Value).Intensity > 0)
-                ActiveEffects.Add(GetEffect((EffectType)effect.Value));
+              if (effect.Value).Intensity > 0)
+                ActiveEffects.Add(effect.Value));
```

Comment:
This won't compile. There's an extra closing parenthesis after each `effect.Value`.